plugins/catalog-react/src/components/InspectEntityDialog/InspectEntityDialog.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity } from '@backstage/catalog-model';
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  makeStyles,
+  Tab,
+  Tabs,
+} from '@material-ui/core';
+import React, { useEffect } from 'react';
+import { AncestryPage } from './components/AncestryPage';
+import { ColocatedPage } from './components/ColocatedPage';
+import { JsonPage } from './components/JsonPage';
+import { OverviewPage } from './components/OverviewPage';
+import { YamlPage } from './components/YamlPage';
+
+const useStyles = makeStyles(theme => ({
+  fullHeightDialog: {
+    height: 'calc(100% - 64px)',
+  },
+  root: {
+    display: 'flex',
+    flexGrow: 1,
+    width: '100%',
+    backgroundColor: theme.palette.background.paper,
+  },
+  tabs: {
+    borderRight: `1px solid ${theme.palette.divider}`,
+    flexShrink: 0,
+  },
+  tabContents: {
+    flexGrow: 1,
+    overflowX: 'auto',
+  },
+}));
+
+function TabPanel(props: {
+  children?: React.ReactNode;
+  index: any;
+  value: any;
```

Comment:
does these have to be any?

Reply:
Oh good catch, this was actually copy pasted from the mui docs :) Will fix